lib/raygun.offline.ts

```diff
@@ -29,15 +28,14 @@ export class OfflineStorage implements IOfflineStorage {
   }
 
   private _sendAndDelete(item: string) {
-    const storage = this;
-
-    fs.readFile(path.join(this.cachePath, item), "utf8", function (
-      err,
-      cacheContents
-    ) {
-      storage.transport.send(cacheContents);
-      fs.unlink(path.join(storage.cachePath, item), () => {});
-    });
+    fs.readFile(
+      path.join(this.cachePath, item),
+      "utf8",
+      (err, cacheContents) => {
+        this.transport.send(cacheContents);
```

Comment:
Do we need to check if the transport has worked before deleting? Maybe, put the delete in a promise, if send succeeds?

Reply:
The package functionality has not been changed in this case. I am currently working on adding async/await support to the `send` call, which would make easier to do this check, otherwise I'd have to use a callback.

Can we wait for the next release to do this?

Reply:
Resolution: Agreement on looking at this issue again in the context of the async/await changes.

Reply:
To be looked at again in the context of https://github.com/MindscapeHQ/raygun4node/issues/146